src/state/actions/__test__/companies.test.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import * as rtl from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import thunk from 'redux-thunk';
+import * as types from '../../types/index';
+
+import { getCompanies, addCompany } from '../companies';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+const mock = new MockAdapter(axios);
+
+const testCompanies = [
+  {
+    id: 1,
+    name: 'Accenture',
+    website: 'www.accenture.com.',
+    location: 'Atlanta, GA',
+    type: 'Business',
+    logo: '',
+    description: '',
+    created_at: null,
+    updated_at: null,
+    latitude: 33.7537,
+    longitude: -85,
+    average_rating: '4.00000000000000000000',
+  },
+];
+
+const testNewCompany = {
+  name: 'Accenture',
+  website: 'www.accenture.com.',
+  location: 'Atlanta, GA',
+  longitude: -85.0,
+  latitude: 33.7537,
+  type: 'Business',
+  logo: '',
+  description: '',
+};
+
+afterEach(rtl.cleanup);
+let wrapper;
+beforeEach(
+  rtl.cleanup
+  //   () => {
+  //   wrapper = rtl.render;
+  // }
```

Comment:
If you're not using a code block, it's best you remove it.

Reply:
Fixed.